system/assets/js/frontend/formbuilder/datePicker/bootstrap.datepicker.js

```diff
@@ -1038,9 +1038,9 @@
 		},
 		headTemplate: '<thead>'+
 							'<tr>'+
-								'<th class="prev"><i class="fa fa-arrow-left"/></th>'+
+								'<th class="prev"><i class="font-icon font-icon-left"/>&lsaquo;</th>'+
```

Comment:
Do these classes work in raw Preside? I guess as this is the frontend it may not matter. But this could break existing implementations if they are already built to work with these classes?

Reply:
Currently there is no working CSS classes for both `font-icon-*` (_core) or `fa-*` in basic Preside. Agree we stick with modified original `fa-*` to prevent breaking classes.